CHANGELOG.md

```diff
@@ -116,111 +118,6 @@ In this release, we have added support for most of the OneView settings resource
 - oneview_uplink_set_facts
 - oneview_version_facts
 
-## v6.0.0
```

Comment:
revert these changes

Reply:
in master branch, ver6.0.0 is added twice, hence removed.